rse-teams.Rmd

```diff
@@ -442,6 +395,38 @@ because there's no way to retire a label without deleting it.
 After a while,
 a project that uses labels for specific events can have a *lot* of old labels…
 
+## How do we decide the priority of certain tasks? {#rse-teams-prioritize}
+
+Their purpose is to help [triage][triage],
+which is the process of deciding what is going to be worked on in what order.
+
+In a large project,
+triage is the responsibility of a [product manager][product-manager].
+In a small one,
+the project's lead usually makes triage decisions,
+or project members add [up-votes][up-vote] and [down-votes][down-vote] to issues
+to indicate how important they think things are.
+Regardless of who's responsibility
+
```

Comment:
add: 
assigning priority to tasks directly relates to your project planning approach. It is advised to adopt Agile planning style for software development projects.